extra/linux/io.alacritty.Alacritty.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016-2019 Joe Wilm, The Alacritty Project Contributors -->
+<component type="desktop-application">
+  <id>io.alacritty.Alacritty</id>
+  <!-- Translators: The application name -->
+  <name>Alacritty</name>
+  <launchable type="desktop-id">io.alacritty.Alacritty.desktop</launchable>
+  <project_license>APACHE-2.0</project_license>
+  <metadata_license>APACHE-3.0</metadata_license>
```

Comment:
Why did you pick APACHE-3.0 here?

Reply:
I scoured your code to glean info. I thought I saw that in relation to some metadata, but I can't find it now. Would you prefer I amend to `APACHE-2.0`? Change made.